tests/build/build_templates.go

```diff
@@ -40,20 +41,22 @@ const pipelineCompletionRetries = 2
 
 // OnboardComponent onboards a component from a test repository URL and returns the component's name
 func OnboardComponent(ctrl *has.HasController, gitUrl, revision, applicationName, componentName, namespace string) string {
```

Comment:
Now it's more of createComponent function.
Does it make sense to do actual onboarding in this function?

Reply:
Could you please explain a bit more.. by onboarding do you mean creatingComponent + something?

Reply:
Yes, that sounds so to me (creation is even optional in this case).
And since we only create a Component in that function, we should name it `createComponent` or do actual onboarding in it.

Reply:
Got it.. renamed it to `CreateComponent` now